bitnami/common/templates/_utils.tpl

```diff
@@ -65,3 +65,13 @@ Usage:
 {{- end -}}
 {{- printf "%s" $key -}} 
 {{- end -}}
+
+{{/*
+Checksum a template containing a *single* resource (ConfigMap,Secret) for use in pod annotations, excluding the metadata (see #18376).
+Usage:
+{{ include "common.utils.checksum" (dict "template" "/configmap.yaml" "context" $) }}
+*/}}
+{{- define "common.utils.checksum" -}}
```

Comment:
To be in sync with other functions defined here, I'd use a different name for the function (getSHA256sum, getTemplateSHA256sum, ... ) to describe better what the function does. WDTY?

Reply:
let's try [checksum](https://en.wiktionary.org/wiki/checksum#Verb)Template (sha256 is an implementation detail and does not need to be exposed).
The symmetry in *checksum*/xxx et common.utils.*checksumTemplate*  looks neat and it makes the lines around 120 chars.
```yaml
      annotations:
        checksum/health: {{ template "common.utils.checksumTemplate" (dict "path" "/health-configmap.yaml" "context" .) }}
        checksum/scripts: {{ template "common.utils.checksumTemplate" (dict "path" "/scripts-configmap.yaml" "context" .) }}
```

Reply:
Great! That's a good name for the function too! 👍